jp/components/grid_groupby.md

```diff
@@ -127,6 +127,13 @@ export interface IGroupByRecord {
 |--- |--- |
 |グループ列の最大値は 10 です。 | 10 列以上の場合はエラーがスローされます。
 
+### API
+
+* [IgxGridComponent]({environment:angularApiUrl}/classes/igxgridcomponent.html)
+* [ISortingExpression]({environment:angularApiUrl}/interfaces/isortingexpression.html)
+* [IgxColumnComponent]({environment:angularApiUrl}/classes/igxcolumncomponent.html)
+* [IGroupByExpandState]({environment:angularApiUrl}/interfaces/igroupbyexpandstate.html)
+
```

Comment:
Could provide Typedoc and Sassdoc references for IgxChipCompoenent and igx-chip-theme, respecrively

Reply:
@HristoP96 Please check the changes that were made.